pkg/network/driver/common_test.go

```diff
@@ -95,6 +95,21 @@ var _ = Describe("DhcpConfig", func() {
 			vif := createDummyVIF(vifName, ipv4Cidr, ipv4Gateway, ipv6Cidr, mac, mtu)
 			Expect(vif.String()).To(Equal(fmt.Sprintf("DhcpConfig: { Name: %s, IP: %s, Mask: %s, IPv6: %s, MAC: %s, AdvertisingIPAddr: %s, MTU: %d, IPAMDisabled: false}", vifName, ipv4Address, ipv4Mask, ipv6Cidr, mac, ipv4Gateway, mtu)))
 		})
+		It("returns correct string representation when an IP is not defined", func() {
+			macAddr, _ := net.ParseMAC(mac)
+			dhcpConfig := cache.DhcpConfig{
```

Comment:
Please use `createDummyDhcpConfig` as the other 2 tests do.

Reply:
I cannot; the bug happens when you don't pass it an IP. 
It would panic in https://github.com/kubevirt/kubevirt/blob/3d6d7bc2b842c1a406932cf30e2e7f3ead3b449a/pkg/network/driver/common_test.go#L122

Reply:
Kind of facelifted the helper function to provide a more common set of expectations when creating the `dhcpConfig` objects.

Is it better now ?